documentation/c04-useful.sil

```diff
@@ -317,4 +317,37 @@ This means that for the environment block version (\code{\\begin\{lua\}}) there
 The first instance of such an end tag terminates the block, and there is currently no way to escape it.
 For the inline command form (\code{\\lua}) an exact matching number of open and closing braces may appear in the Lua code—the next closing brace at the same level as the opening brace will close the tag, even if it happens to be inside some quoted string in the Lua code.
 Including any number of nested opening and closing braces is possible as long as they are balanced.
+
+\section{Including raw inline content}
+
+When parsing a SIL file, SILE invokes an “inputter” module, which implements the SIL language grammar and constructs an abstract syntax tree (AST) for processing.
+This implies that the content of any command or environment is in SIL syntax.
+
+However, there are cases when you may need to pass raw content that should remain unparsed — or, more properly, later parsed by a \em{different} grammar.
+While you could escape all special characters in your content with backslashes to prevent them from being interpreted as SIL constructs, this approach is tedious and cumbersome.
+
+This issue already arises in several scenarios.
+For instance, the \code{\\lua} command (and the legacy \code{\\script} command) described above fall into this category.
+In these cases, one expects to use Lua code without the need for escaping it.
+
+Similarly, the content of the \code{\\math} command (for the \autodoc:package{math} package) falls outside the scope of the SIL language syntax and requires a different grammar.
+After all, its content follows the TeX math syntax, with commands with multiple arguments, special use of brackets, and so on.
+Therefore, we need to instruct the SIL parser that this content should not be interpreted,but rather extracted as a raw string.
+Later, it will be fed to another dedicated inputter for parsing.\footnote{In the case of math, it is currently a \em{pseudo}-inputter, but that is an implementation detail.}
+
+The SIL inputter reserves a few special keywords: \code{\\lua}, \code{\\script}; but also \code{\\ftl}, \code{\\math}, \code{\\sil}, \code{\\use}, \code{\\xml}; and finally \code{\\raw}, which we will discuss here.
+
+It is obvious that we can’t reserve too many keywords in advance.
+However, they must be known \em{before} parsing a file, which means they can’t be dynamic.
+The reserved keywords can’t be overridden or redefined afterwards.
```

Comment:
```suggestion
The reserved keywords can’t be overridden or redefined after document parsing has begun.
```

It is actually possible to add keywords and define your own, you just have to do it before parsing starts. The important thing is the dynamic nature can't be defined inside the document (or any module loaded by the document of course) it has to be done on the CLI or some other way. And there are ways. We don't have to explain those use cases, but I do think clarifying what the limitation actually is is worth it.

To my mind saying "afterwards" leaves open the question of "during".